@xen-orchestra/backups/writers/_AbstractWriter.js

```diff
@@ -0,0 +1,5 @@
+exports.AbstractWriter = class AbstractWriter {
```

Comment:
Add a constructor which will receive params passed to all writers and store them on the instance.